### DISCUSSION TITLE
Docker image

### DISCUSSION THREAD

**Discussion by habsfanongit** [crewAIInc/crewAI#1327] [General]
While trying to containerize an agent, I found this docker image I wanted to share with others https://github.com/sageil/crewai-docker-image. The image includes a preconfigured lazyvim using latest neovim.

**Comment by hqnicolas** [comment]
It is not possible to know what is inside this image, the files present in github are obfuscated, this repository is very strange, I will make a safe fork

https://github.com/sageil/crewai-docker-image/pull/1

**Comment by sageil** [comment]
Thanks @habsfanongit 

@hqnicolas , I am the author of the repo. Would you mind sharing with us what do not you consider as safe in this image?  Also, None of the files in the repo is obfuscated. 

You can see everything inside the image by comparing what's in the docker file in the repo and viewing the image layers on the image tag https://hub.docker.com/layers/sageil/crewai/latest/images/sha256-f18f8f7fb053ff474763a7d4deef1432c5bd60614576f4c8573eedc77ab0cae0?context=explore

**Comment by hqnicolas** [comment]
@sageil
you need to let users use your own images
I have seen many projects receive malicious updates and not including this file in their repository causes users to lose control.
https://github.com/hqnicolas/crewai-docker-image/tree/main/build
as you said, Included changes are personal preference. Please use your own fork to modify the readme file.

**Reply by sageil** [reply]
@hqnicolas, maybe there is a communication issue here. 

User can build and use their own images, there is nothing in the repo to prevent anyone from building the image. If you have any questions about the steps used to build the image, I am more than happy to answer them.

Finally, I agree.  No one should use anything just because its published. Always do your due diligence because you never know others intentions. 

Thank you and please if you add an improvement, share with the community.
